src/inmanta/server/config.py

```diff
@@ -278,19 +278,48 @@ def validate_fact_renew(value: object) -> int:
 # Dashboard
 #############################
 
-dash_enable = Option("dashboard", "enabled", True, "Determines whether the server should host the dashboard or not", is_bool)
+dash_enable = Option(
+    "dashboard",
+    "enabled",
+    True,
+    "[DEPRECATED USE :inmanta.config:option:`web-ui.dashboard_enabled`] "
```

Comment:
What's the best way to check how it renders ? (I tried to build it locally with `make docs` but it seems to fail on a couple of places e.g ```\sphinxincludegraphics[]{None}``` )

And I can't get it to build either on master, I might be missing something

Reply:
The easiest way is to use the `build-docs` command from irt.

* First build the documentation from master by setting the `branch` and `build-type` arguments to `master` and `dev` respectively
* Then change the branch checked out into the `output-dir` to your branch
* Perform the build again but this time with the `--in-place` option to make sure that your manual changes in the output-dir remain untouched.

Reply:
I think I'm still missing something, since I can't test the full build with both my branches (on inmanta-core and inmanta-ui), I'm not 100% that the web-ui section is visible in the doc.
For instance when I make docs on inmanta-core, "[DEPRECATED USE :inmanta.config:option:`web-ui.dashboard_enabled`] " renders properly but there is no clickable link unlike other similar deprecated options.

Reply:
This bug https://github.com/inmanta/irt/issues/1409 is probably the reason why the link to the config option doesn't work. Can you try again by:
* Manually installing the components you changed in the output directory of the product build into the venv used by the product build. (This is the venv present in the output directory)
* Then, perform the `build-docs` command again using the `--in-place` option.

If you feel like it, you can also just fix the bug first. That's up to you.

Reply:
It does indeed work this way :+1: